File: Easy/remove_element.php
<?php
class Solution {

    /**
     * @param Integer[] $nums
     * @param Integer $val
     * @return Integer
     */
    function removeElement($nums, $val) {
        return count(array_filter($nums, function($v) use ($val){
            return $v !== $val;
        }));
    }
}

$solution = new Solution;

print_r($solution->removeElement([3, 2, 2, 3], 3));
print_r($solution->removeElement([0, 1, 2, 2, 3, 0, 4, 2], 2));

File: Easy/remove_element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        return len([v for v in nums if v != val])
        
print(Solution().removeElement([3, 2, 2, 3], 3))
print(Solution().removeElement([0, 1, 2, 2, 3, 0, 4, 2], 2))
        